### ISSUE TITLE
Catchable fatal error: Argument 1 passed to wapmorgan\PhpCodeFixer\PhpCodeFixer::delimByComma() must be of the type array, null given

### ISSUE THREAD

**Issue by websmurf** [wapmorgan/PhpCodeFixer#11]
Hi

I'm trying to use your project on a legacy code base and I'm running into the following error: 

```
Max file size set to: 1.000 MiB
Scanning app ...

Catchable fatal error: Argument 1 passed to wapmorgan\PhpCodeFixer\PhpCodeFixer::delimByComma() must be of the type array, null given, called in /Users/adam/.composer/vendor/wapmorgan/php-code-fixer/data/password_hash_salt_option.php on line 6 and defined in /Users/adam/.composer/vendor/wapmorgan/php-code-fixer/src/PhpCodeFixer.php on line 213
```

This only happens when I set 7.0 or 7.1 as target, when using 5.6 or lower it works fine.

My PHP version:
```
$ php -v
PHP 7.0.15 (cli) (built: Feb 16 2017 15:05:51) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
    with Xdebug v2.5.1, Copyright (c) 2002-2017, by Derick Rethans
```

**Comment by wapmorgan** [comment]
I need additional information to fix this problem. Can you run for every folder in your project and find fine which breaks phpcf?

**Comment by wapmorgan** [comment]
Or wait until I add debug output functionality in `phpcf`.

**Comment by websmurf** [comment]
Good call, there is only one subfolder where it breaks on, and after that I've narrowed it down to a single file, our Password.php class.

In relation to the error above, this is the relevant piece I think:
```
password_hash($password, PASSWORD_DEFAULT);
```

Could it be because we are not providing the 3rd (optional) argument there?

**Comment by wapmorgan** [comment]
Yes, it is. But this is wrong behavior and fixed in **2.0.4**. Upgrade phpcf and have a nice day.

**Comment by websmurf** [comment]
Jup, fixed. Thanks for the quick response and fix!
Have a good day too!
